acquamanCommon.pri

```diff
@@ -467,9 +467,9 @@ HEADERS += \
 	source/ui/beamline/AMSlitView.h  \
 	source/beamline/AMSlits.h \
 	source/ui/beamline/AMSlitsView.h \
-	source/beamline/AMTemperatureMonitor.h \
-	source/beamline/AMTemperatureMonitorGroup.h \
-	source/beamline/AMTemperatureMonitorGroupStatus.h
+    $$PWD/source/beamline/AMBeamlineControlGroupStatus.h \
+    $$PWD/source/beamline/AMBeamlineControlGroup.h \
+    $$PWD/source/beamline/AMBeamlineControl.h
```

Comment:
I'm wondering whether the `$$PWD/` is mandatory. If not, I prefer to remove it to be consistent with the other header/source files.